Source/effects.cpp

```diff
@@ -1413,8 +1413,14 @@ void effects_play_sound(const char *snd_file)
 
 int GetSFXLength(int nSFX)
 {
+#ifndef DISABLE_STREAMING_SOUNDS
+	constexpr bool kAllowStreaming = true;

```

Comment:
Plz respect the code style, it's only on commit old :D

```suggestion
	constexpr bool allowStreaming = true;
```

Reply:
Done